spec/lib/geocode_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe Geocode do
+  it 'normalises the postcode' do
+    expect(Geocoder).to receive(:coordinates).with('RG11GG, United Kingdom')
+
+    Geocode.call('rg1 1gg')
+  end
+
+  it 'returns the lat/long pair' do
+    VCR.use_cassette(:rg11gg) do
+      latitude, longitude = Geocode.call('RG11GG')
+
+      expect(latitude).to  be
+      expect(longitude).to be
```

Comment:
@esbel what you're seeing is parallel assignment, it destructures any arrays returned from `Geocoder.coordinates(*)`. so this code is equivalent to:

```ruby
result = Geocode.call(...)
expect(result[0]).to be
expect(result[1]).to be
```